examples/advanced/xgboost/histogram-based/README.md

```diff
@@ -33,7 +33,7 @@ tensorboard --logdir /tmp/nvflare/xgboost_v2_workspace/simulate_job/tb_events
 
 ## Timeout configuration
 
-Please refer to [Federated XGBoost Timeout Mechanism](https://nvflare.readthedocs.io/en/2.4/user_guide/federated_xgboost/timeout.html)
+Please refer to [Federated XGBoost Timeout Mechanism](https://nvflare.readthedocs.io/en/2.4/user_guide/federated_xgboost/reliable_xgboost_timeout.html)
```

Comment:
link should change 2.4 -> main? we could prob. move these link update in another PR.

Reply:
Right I can update the links once we move the 2.4 work to main. The link change here is needed to pass the premerge check.

Reply:
Ah I did not notice this PR is against 2.4